app/components/anchor/radio_buttons_component.rb

```diff
@@ -0,0 +1,139 @@
+module Anchor
+  class RadioButtonsComponent < Component
```

Comment:
_non-blocking:_ Maybe renaming this to help differentiate that from the singular component?
```suggestion
  class RadioButtonCollectionComponent < Component
```

Reply:
Great idea. Will do.